matRad_calcParticleDoseMCsquare.m

```diff
@@ -189,12 +189,12 @@
 bdFile = [machine.meta.machine '.txt'];
 
 % bdFile = 'BDL_matRad.txt'; %use for baseData fit 
-MCsquareBDL = MatRad_MCsquareBaseData(machine,stf);
+MCsquareBDL = MatRad_MCsquareBaseData(machine, stf);
 %matRad_createMCsquareBaseDataFile(bdFile,machine,1);
 % MCsquareBDL = MCsquareBDL.saveMatradMachine('test');
 MCsquareBDL = MCsquareBDL.writeMCsquareData([MCsquareFolder filesep 'BDL' filesep bdFile]);
 %movefile(bdFile,[MCsquareFolder filesep 'BDL/' bdFile]);
-% MCsquareBDL = MCsquareBDL.saveMatradMachine('testMachine');
+% MCsquareBDL = MCsquareBDL.saveMatradMachine('savedMatRadMachine');
```

Comment:
Is it possible to remove some of the comments here that are not necessary?

One more idea: We could automatically save a mat-file with the emittance if it is automatically approximated (along with an appropriate mesesage that we approximated the phase space and that the machine is saved into a matfile which could have a name like, e.g., modality_machineName_lastMCApproximation. What do you think?